b2sdk/encryption/setting.py

```diff
@@ -159,12 +182,15 @@ def add_key_id_to_file_info(self, file_info: Optional[dict]):
             return file_info
         if file_info is None:
             file_info = {}
-        if file_info.get(SSE_C_KEY_ID) is not None and file_info[SSE_C_KEY_ID] != self.key.key_id:
+        if file_info.get(SSE_C_KEY_ID_FILE_INFO_KEY_NAME) is not None and file_info[
+            SSE_C_KEY_ID_FILE_INFO_KEY_NAME] != self.key.key_id:
             raise ValueError(
-                'Ambiguous key id set: "%s" in file_info and "%s" in %s' %
-                (file_info[SSE_C_KEY_ID], self.key.key_id, self.__class__.__name__)
+                'Ambiguous key id set: "%s" in file_info and "%s" in %s' % (
+                    file_info[SSE_C_KEY_ID_FILE_INFO_KEY_NAME], self.key.key_id,
+                    self.__class__.__name__
+                )
             )
-        file_info[SSE_C_KEY_ID] = self.key.key_id
+        file_info[SSE_C_KEY_ID_FILE_INFO_KEY_NAME] = self.key.key_id
```

Comment:
here you are doing a dict get operation three times. File info is fairly limited as of now, so maybe that's not a problem, but generally you should assign it to a temporary variable and then compare to none and to another key (and put in the error). It's a minor issue.